testnets/epixtestnet/chain.json

```diff
@@ -0,0 +1,109 @@
+{
+  "$schema": "../../chain.schema.json",
+  "chain_name": "epixtestnet",
+  "status": "live",
+  "network_type": "testnet",
+  "website": "https://epix.zone/",
+  "pretty_name": "Epix Testnet",
+  "chain_type": "cosmos",
+  "chain_id": "epix_4243-0",
+  "bech32_prefix": "epix",
+  "daemon_name": "epixd",
+  "node_home": "$HOME/.epixd",
+  "key_algos": [
+    "ethsecp256k1"
+  ],
+  "extra_codecs": [
+    "ethermint"
+  ],
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "atepix",
+        "low_gas_price": 20000000000,
+        "average_gas_price": 25000000000,
+        "high_gas_price": 40000000000
+      }
+    ]
+  },
+  "staking": {
+    "staking_tokens": [
+      {
+        "denom": "atepix"
+      }
+    ]
+  },
+  "codebase": {
+    "git_repo": "https://github.com/EpixZone/Epix",
+    "recommended_version": "v12.0.0-rc4",
+    "compatible_versions": [
+      "v12.0.0-rc4"
+    ],
+    "cosmos_sdk_version": "0.46",
```

Comment:
please add the cosmos_sdk, consensus, and ibc version data into the v1 object in the versions array.
Note that these legacy properties will also require the replacement properties (e.g, cosmos_sdk_version -> sdk: type, version; etc.)

Reply:
Fixed